packages/blockchain/test/reorg.spec.ts

```diff
@@ -64,23 +58,6 @@ tape('reorg tests', (t) => {
         'low TD block should have a higher number than high TD block'
       )
 
-      await blockchain.putBlocks(blocks_lowTD)
-
-      const head_lowTD = await blockchain.getHead()
-
-      await blockchain.putBlocks(blocks_highTD)
-
-      const head_highTD = await blockchain.getHead()
-
-      t.ok(
-        head_lowTD.hash().equals(lowTDBlock.hash()),
-        'head on the low TD chain should equal the low TD block'
-      )
-      t.ok(
-        head_highTD.hash().equals(highTDBlock.hash()),
-        'head on the high TD chain should equal the high TD block'
-      )
-
```

Comment:
Hi @ScottyPoi, what is the reason for this test cases to be removed? Couldn't this be replaced by the newly named function calls?

Won't make this a blocker here so that we do not have another iteration, but in case that you would think this could be kept it would be nice if you submit a small follow-up PR. Thanks! 🙂